FreeRTOS_DNS.c

```diff
@@ -47,11 +47,11 @@
 #include "NetworkBufferManagement.h"
 #include "NetworkInterface.h"
 
-#include "DNS/DNS_Globals.h"
-#include "DNS/DNS_Cache.h"
-#include "DNS/DNS_Parser.h"
-#include "DNS/DNS_Networking.h"
-#include "DNS/DNS_Callback.h"
+#include "DNS_Globals.h"
+#include "DNS_Cache.h"
+#include "DNS_Parser.h"
+#include "DNS_Networking.h"
+#include "DNS_Callback.h"
```

Comment:
@alfred2g, good job, thank you.  It compiled perfectly and all DNS-related tests went well.